### DISCUSSION TITLE
Git tab not updating

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3064] [Q&A]

**Comment by jsjoeio** [comment]
Hey! Do you mind providing a bit more information? Or if this is a bug, feel free to open a bug report!

**Reply by ToasterSticks** [reply]
Sure, It only updates after clicking that reload button, or reloading the page. Is there a way to make it automatic? The discussion title was quite misleading...

**Reply by jsjoeio** [reply]
Hmm...I would need more context and steps to reproduce (maybe a video too). Do you mind opening up a bug report?

**Reply by ToasterSticks** [reply]
Wondering if this should be automated, as that is how it works on my desktop version of VSCode 

https://user-images.githubusercontent.com/64027609/113922216-072a0080-97b5-11eb-8b0f-3b6ebc6f5620.mov

**Reply by jsjoeio** [reply]
Ahhh! I see now. Yeah, this is probably a bug in code-server then. Do you mind opening a bug report [here](https://github.com/cdr/code-server/issues/new?assignees=&labels=&template=bug-report.md&title=)?
